public/app/features/live/centrifuge/service.ts

```diff
@@ -172,11 +199,18 @@ export class CentrifugeSrv {
           }
         }
 
-        const elapsed = liveTimer.lastUpdate - last;
-        if (elapsed > 1000 || liveTimer.ok) {
+        if (this.dataStreamSubscriberReady) {
```

Comment:
`elapsed > 1000` not sure how essential is this condition - can we do without it? maybe we can go with something simpler, such as  _send if we have not sent any message for the past X (5?) seconds for this channel_ @ryantxu

Reply:
Open to suggestions ;)  the 1s was just an arbitrary condition to make sure things keep rendering even when the timer can not keep up.  In some cases (like what @atifali showed) "ok" is never true